src/main/java/org/elasticsearch/cluster/service/InternalClusterService.java

```diff
@@ -310,6 +310,12 @@ public void run() {
         return pendingClusterTasks;
     }
 
+    @Override
+    public int numberOfPendingTasks() {
```

Comment:
we have `numberOfPendingTasks` and `getNumberOfPendingTasks` can we find a consistent naming?